rsmtool/rsmsumm.py

```diff
@@ -0,0 +1,205 @@
+"""
+The script to create a comparison report for multiple models
+
+:author: Anastassia Loukina (aloukina@ets.org)
+:author: Nitin Madnani (nmadnani@ets.org)
+:organization: ETS
+"""
+
+#!/usr/bin/env python
+
+import argparse
+import glob
+import logging
+import os
+import sys
+
+from os.path import abspath, dirname, exists, join, normpath
+
+
+from rsmtool.input import (check_main_config,
+                           locate_file,
+                           read_json_file,
+                           locate_custom_sections)
+
+from rsmtool.report import (create_summary_report,
+                            get_ordered_notebook_files)
+
+
+from rsmtool.utils import LogFormatter
+
+from rsmtool.version import __version__
+
+
+def check_experiment_dir(experiment_dir, configpath):
+    """
+    Check that the supplied experiment directory exists and contains
+    the output of the rsmtool experiment.
+
+    Parameters
+    experiment_dir : str
+        Supplied path to the experiment_dir
+    configpath: str
+        Path to the directory containing the configuration file
+
+    Returns
+    -------
+    jsons : list
+        A list paths to all configuration json files contained in the output directory
+
+    Raises
+    ------
+    FileNotFoundError
+        If the directory does not exist or does not contain and output
+        of an RSMTool experiment
+    """
+
+    full_path_experiment_dir = locate_file(experiment_dir, configpath)
+    if not full_path_experiment_dir:
+        raise FileNotFoundError("The directory {} "
+                                "does not exist.".format(experiment_dir))
+    else:
+        # check that there is an output directory
+        csvdir = normpath(join(full_path_experiment_dir, 'output'))
+        if not exists(csvdir):
+            raise FileNotFoundError("The directory {} does not contain "
+                                    "the output of an rsmtool "
+                                    "experiment.".format(full_path_experiment_dir))
+
+        # find the json config files  for all experiments stored in this directory
+        jsons = glob.glob(join(csvdir,  '*.json'))
+        if len(jsons) == 0:
+            raise FileNotFoundError("The directory {} does not contain "
+                                    "the .json configuration files for rsmtool "
+                                    "experiments.".format(full_path_experiment_dir))
+
+    return jsons
```

Comment:
This should be indented to be under `else` or `jsons` should be defined to be an empty list for the `if` case.